### ISSUE TITLE
Help translate Mutify.

### ISSUE THREAD

**Issue by teekamsuthar** [teekamsuthar/Mutify#3]
Although, translations to any language are welcomed. Here are the translations required as per user-base ordered by priority -

- [ ] German (Germany)
- [ ] Spanish (United States)
- [ ] French (France)
- [ ] Vietnamese (Vietnam)
- [ ] German (Austria)
- [ ] Spanish (Spain)
- [ ] Spanish (Latin America)

Recommend tool for translation - Android Studio with translation helper plugin.

**Comment by gupta-shrinath** [comment]
Hi, Teekam Congratulations on building this helpful application. I would like to work on this. I have translated it to German(Germany) and Spanish (Spain) using google translate and working on Vietnamese and French. The `dialog_positive_btn` and `open_spotify`string key have the same value Open Spotify should I change code to use only `open_spotify`?

**Comment by gupta-shrinath** [comment]
The source code in the master branch seems to be of the older version of the application. Maybe you have continued the development of the application in private. If the current version of the application doesn't support the languages mentioned in the PR you could merge it because it won't have any major breaking changes.

**Comment by teekamsuthar** [comment]
Thanks for helping out with the current translations in Mutify @gupta-shrinath, I will be pushing out a new commit with updated strings soon, so please let me know if you will be able to help with translation of those as well. :)

**Comment by teekamsuthar** [comment]
>  The `dialog_positive_btn` and `open_spotify`string key have the same value Open Spotify should I change code to use only `open_spotify`?

Yes, please keep all the extra duplicate values, even if they're same. The id's are different so that these can be identified easily in code, also it helps with code readability. Thanks :)

**Comment by gupta-shrinath** [comment]
> > The `dialog_positive_btn` and `open_spotify`string key have the same value Open Spotify should I change code to use only `open_spotify`?
> 
> Yes, please keep all the extra duplicate values, even if they're same. The id's are different so that these can be identified easily in code, also it helps with code readability. Thanks :)
>
Make sense. I thought of this and to be on the safer side created a new [commit](https://github.com/teekamsuthar/Mutify/pull/6/commits/ea37d9185257a89687f5cffb99bef3dff3a81c07)  I'll just have to revert the commit in the PR.


> Thanks for helping out with the current translations in Mutify @gupta-shrinath, I will be pushing out a new commit with updated strings soon, so please let me know if you will be able to help with translation of those as well. :)
>
Sure.

**Comment by gupta-shrinath** [comment]
Hi, @teekamsuthar I have reverted the [commit](https://github.com/teekamsuthar/Mutify/pull/6/commits/ea37d9185257a89687f5cffb99bef3dff3a81c07) which removes the key `dialog_positive_btn`.
